charts/nexus-iq/README.md

```diff
@@ -93,6 +93,9 @@ The command removes all the Kubernetes components associated with the chart and
 
 You can define the `config.yml` for IQ Server in your `myvalues.yml` file on startup. 
 It is the `iq.configYaml` property. For more details, see the [Configuring IQ Server](https://help.sonatype.com/iqserver/configuring) help page.
+Additionally the server can be started with JAVA_OPTS exported to the environment. This will be added to the server 
+process invocation and can be used for purposes such as changing the server memory settings. See the defaults set in
+[the values.yml file](values.yml).
```

Comment:
Is this the `iq.env` thing? Should it be added to the Chart Configuration Options table above? I do kind of like the suggestion of seeing the values.yaml for all defaults, so that it doesn't need to be documented (and updated) in multiple places. 👍

Reply:
yeah, added the more general `iq.env` reference above